common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -66,6 +66,13 @@ class SequenceFields(object):
         scope=Scope.settings,
     )
 
+    self_paced = Boolean(
+        display_name=_("Course is self-paced"),
+        help=_("ask Eric to remove this field"),
+        default=False,
+        scope=Scope.settings,
+    )
```

Comment:
?  Since it is defined in inheritance.py, shouldn't it be available to the sequence block?